README.md

```diff
@@ -24,6 +24,45 @@ oriented architecture.
 
 Dashboard datamodel is still under review.
 
+### Roadmap
+
+Here an overview of what we are going to implement / provide in a short term
+
+#### First Milestone - target Beginning of December - Dashboard management
```

Comment:
Hmm, might be a bit bold to include timelines in here?

Reply:
mmm don't know :D. Can be interesting to have it at this time anyway haha

Reply:
I removed it